crypto/doc/CHANGELOG_cc3xx_mbedcrypto.rst

```diff
@@ -9,6 +9,46 @@ Changelog - nrf_cc3xx_mbedcrypto
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_mbedcrypto - 0.9.19
+*****************************
+
+New version of the runtime library with the following bug fixes and improvements:
+
+* Fixed EdDSA error return-codes
+* Don't define duplicate marcros MBEDTLS_ECP_MAX_BYTES and MBEDTLS_ECP_MAX_PT_LEN
+* Update to mbed TLS 3.5.2
+
+Library built against Mbed TLS version 3.5.2.
+
+This version is dependent on the nrf_cc310_platform or nrf_cc312_platform library for low-level initialization of the system and proper RTOS integration.
+
+Added
+=====
+
+Added a new build of nRF_cc3xx_mbedcrypto libraries for nRF91 Series, nRF52840, and nRF5340.
+
+.. note::
+
+   The *short-wchar* libraries are compiled with a wchar_t size of 16 bits.
+
+* nrf_cc312_mbedcrypto, nRF5340 variants
+
+  * :file:`crypto/nrf_cc312_mbedcrypto/lib/cortex-m33/**/libnrf_cc312_psa_crypto_0.9.18.a`
```

Comment:
Wrong library version in this list (says 0.9.18, should be 0.9.19) Repeats for the remainder of this list

```suggestion
  * :file:`crypto/nrf_cc312_mbedcrypto/lib/cortex-m33/**/libnrf_cc312_psa_crypto_0.9.19.a`
```